rules/windows/process_creation/win_remote_powershell_session_process.yml

```diff
@@ -1,6 +1,6 @@
-title: Remote PowerShell Session
+title: Remote PowerShell Session Host Process (WinRM)
 id: 734f8d9b-42b8-41b2-bcf5-abaf49d5a3c8
-description: Detects remote PowerShell sections by monitoring for wsmprovhost as a parent or child process (sign of an active ps remote session)
+description: Detects remote PowerShell sections by monitoring for wsmprovhost (WinRM host process) as a parent or child process (sign of an active ps remote session)
 status: experimental
 date: 2019/09/12
 modified: 2019/11/10
```

Comment:
i think you can edit the modified line with the date of your change 

like this 

```
modified: 2021/05/21
```